client/app/bundles/course/achievement/reducers/editPage.js

```diff
@@ -0,0 +1,27 @@
+import actionTypes from '../constants';
+
+const initialState = {};
```

Comment:
Could you populate the initial state? Although there's not much harm leaving it blank, it makes it easy to see the shape of the state tree